App.tsx

```diff
@@ -48,7 +49,9 @@ const App: FunctionComponent = () => {
               <PermissionsProvider>
                 <SystemServicesProvider>
                   <ExposureProvider>
-                    <MainNavigator />
+                    <AnalyticsProvider>
```

Comment:
Should this live in the `SettingsStack` if that's the only place we use it?

Reply:
This will soon be used basically everywhere (it's where the function to track analytics events lives) so we'll need access to it throughout the app!